cli/net.go

```diff
@@ -445,9 +445,6 @@ var NetReachability = &cli.Command{
 		}
 
 		fmt.Println("AutoNAT status: ", i.Reachability.String())
-		if i.PublicAddr != "" {
-			fmt.Println("Public address: ", i.PublicAddr)
-		}
```

Comment:
I removed this log (?) output. It was misleading to begin with, a libp2p node doesn't only have a single public address. The correct way to obtain the addresses is to use the `Addrs` method on the host.